projects/sbb-esta/angular-maps/esri-config/src/esri-config/esri-config.module.ts

```diff
@@ -1,12 +1,11 @@
-import { CommonModule } from '@angular/common';
 import { ModuleWithProviders, NgModule } from '@angular/core';
 
 import { ESRI_CONFIG_TOKEN } from './esri-config.token';
 import { EsriConfiguration } from './esri-configuration';
 
 @NgModule({
   declarations: [],
-  imports: [CommonModule]
+  imports: []
 })
 export class EsriConfigModule {
```

Comment:
Would it maybe make sense to move the configuration classes and NgModule into the core submodule, since it is required in all other submodules?
What do you think?

Reply:
I think it makes more sense, to leave `EsriConfigModule` as an own module, because it will grow in future.

Reply:
While I don't quite agree (since this seems to be the **core** configuration), I will accept it.